src/util/StringStream.js

```diff
@@ -73,9 +74,14 @@ class StringStream {
   current(){return this.string.slice(this.start, this.pos)}
   hideFirstChars(n, inner) {
```

Comment:
So now `lineStart` appears to be completely ignored, yet you somehow still left this function in. Why?